tests/test_aliases.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 
-from bambi import load_data, Formula, Model
+import bambi as bmb
```

Comment:
Can you explain what this line is doing in all tests? I undestand that its importing bambi as bmb, but how is the bambi top level being used within the test?